recipe-client-addon/skin/shared/Heartbeat.css

```diff
@@ -0,0 +1,289 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Note that I just copy and pasted this from mozilla-central. I could look through it further if we want, but I don't think that is important right now.